Sources/Carpaccio/ImageMetadata.swift

```diff
@@ -402,6 +456,71 @@ public struct ImageMetadata: Codable {
         let oneTenthPrecisionSeconds = round(s * 10.0) / 10.0
         return "\(oneTenthPrecisionSeconds)s"
     }
+    
+    public var humanReadableFlashMode: String? {
+        guard let fm = self.flashMode else {
+            return nil
+        }
+        
+        switch (fm) {
+            case .unknown:
+                return "unknown"
+            case .noFlash:
+                return "No Flash"
+            case .fired:
+                return "Fired"
+            case .firedNotReturned:
+                return "Fired, Return not detected"
```

Comment:
Thanks! The change you made to separate out the codable vs human readable part makes a lot of sense 👍 

I promise this is the last nitpick I will offer on this: I see the precise capitalisation in this new human readable flash mode still follows the `exiftool` documentation page's exact capitalisation, which is kind of strange combo of sentence cased phrases with comma as the separator. Wonder what you'd feel about `"Fired, return not detected"`, `"Off, did not fire"` etc? I'm not a native English speaker so not sure if this is a thing I'm missing. Also not an especially big deal, let me know what you think.

Reply:
Hey @mz2! I see what you're saying now. English is a bit weird (compared to say, German), in that pretty much any capitalization is valid. The way `exiftool` is capitalizing words is valid, but so is yours. I don't really have a preference either way, so I'll edit it to be like your examples. This is your project after all, so you get the final call!

Reply:
Appreciated, thank you! And Finnish is even weirded, I promise 🤣